src/main.ts

```diff
@@ -224,39 +224,48 @@ export default class LlmShortcutPlugin extends Plugin {
       throw new Error(`LLM Shortcut: Prompt file not found: ${promptFilePath}`);
     }
 
-    const { prompt, options } = await this.parseCommandPromptFromFile(file);
+    const userPromptParams = await this.parseUserPromptFromFile(file);
 
-    if (options?.shouldHandleSelectionOnly) {
```

Comment:
Moved the "shouldHandleSelectionOnly" to the "processLlmRequest" to make it work both with File Prompt and Custom Prompt (now it doesn't take effect because the value is always false by default, but it's necessary for the future #16 implementation)

Reply:
Looks like I missed this, thanks!